zkchannels-contract/zkchannel_smartpy_script.py

```diff
@@ -161,46 +184,52 @@ def custClose(self, custBal, merchBal, revLock, s1, s2):
         # Note that all addresses must be implicit accounts (tz1), not smart contracts
         sp.if merchBal != sp.tez(0):
             sp.send(self.data.merchAddr, merchBal)
-        self.data.merchBal = sp.tez(0)
         self.data.status = CUST_CLOSE
```

Comment:
The lines above really need better documentation. I'm also unclear why we don't have helper functions for:
- verifying a PS signature
- converting mutez to finite field elements
- converting the revocation lock to a finite field element

Reply:
I've added some more documentation. The parts which convert types (mutez -> fr  and  bytes -> fr) are smartpy tricks recommended by the tezos developers so I'm not sure what more to put. For more documentation on the signature verification I'd have to ask Ayo.